hztb-servicemanager/src/main/java/com/svs/hztb/sm/registration/controller/RegistrationController.java

```diff
@@ -13,12 +13,16 @@
 
 import com.svs.hztb.api.sm.model.registration.RegistrationRequest;
 import com.svs.hztb.api.sm.model.registration.RegistrationResponse;
+import com.svs.hztb.common.logging.Logger;
+import com.svs.hztb.common.logging.LoggerFactory;
 import com.svs.hztb.service.UserDataService;
 
 @RestController
 @RequestMapping("/user")
 public class RegistrationController {
 
+	private static final Logger LOGGER = LoggerFactory.INSTANCE.getLogger(RegistrationController.class);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "LOGGER" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://ec2-52-23-152-124.compute-1.amazonaws.com:9000/coding_rules#rule_key=squid%3AS1068)